open_city_profile/settings.py

```diff
@@ -236,3 +237,12 @@
                 "Please create a %s file with random characters to generate your secret key!"
                 % secret_file
             )
+
+# Graphene
```

Comment:
could you please move these settings to before `if "SECRET_KEY" not in locals():`. it's good to keep that secret key generation logic at the end of the file, so that you don't have to scroll over it every time you look for some settings.